crates/node/service/src/actors/derivation.rs

```diff
@@ -30,6 +30,10 @@ where
     pipeline: P,
     /// The latest L2 safe head.
     l2_safe_head: L2BlockInfo,
+    /// A receiver that tells derivation to begin.
+    sync_complete_rx: UnboundedReceiver<()>,
```

Comment:
Can this be a `oneshot`?

Reply:
No because in order to not block on the engine sync check, we spawn the check in a thread which will send over the cloned channel. With a oneshot channel, we can't clone the sender since there would only be one sender allowed.